### ISSUE TITLE
Style  Navbar in homepage

### ISSUE THREAD

**Issue by fridaland** [fridaland/InTheFight#18]
### Homepage Navbar

Adjust Navbar so it sits on top of home banner. See screenshot attached below. 
Ensure nave and logo are on the left, with a link to the index page (or a refresh). 

Nav items on the right:
- Events
- Resources
- Contact

![Screen Shot 2020-10-20 at 8 11 36 AM](https://user-images.githubusercontent.com/40510669/96590668-ea078100-12ab-11eb-9b64-2906bcb62e63.png)

![Screen Shot 2020-10-20 at 8 09 21 AM](https://user-images.githubusercontent.com/40510669/96590457-aad93000-12ab-11eb-8ac3-0ad4cc6e9c6c.png)

**Comment by jmoranin98** [comment]
I would like to work on this

**Comment by Johnisaig** [comment]
Hi @fridaland Can I contribute to this repository?

**Comment by fridaland** [comment]
@Johnisaig  Of course! Thrilled to have you join! Let me know if you have any issues getting set up

**Comment by Johnisaig** [comment]
Hi @fridaland where I can find the nav logo?
![logo](https://user-images.githubusercontent.com/45925383/96912286-b2c7e880-14d4-11eb-8b06-ac6485c1836a.png)

**Comment by fridaland** [comment]
@Johnisaig Great question! I [added some png images](https://github.com/fridaland/InTheFight/commit/2346a7213b5e7998b635972098b0a732b83d9756) under `images/logos`. You should be able to find it under `ring.png`!

**Comment by fridaland** [comment]
@Johnisaig how's it going on this issue? Need any support?
